lms/djangoapps/courseware/tests/test_views.py

```diff
@@ -793,30 +793,6 @@ def test_financial_assistance_form_course_exclusion(
 
         self.assertNotIn(str(course.id), response.content)
 
-    @patch.object(CourseOverview, 'load_from_module_store', return_value=None)
-    def test_financial_assistance_form_missing_course_overview(self, _mock_course_overview):
```

Comment:
After fighting with mock and factory boy a bunch, I couldn't figure out how to actually write this test. I need "enrollment.course" to raise a "DoesNotExist" exception. Maybe someone could help me figure out how to do that?

Reply:
Were you planning to update this test to not use the `CourseEnrollmentFactory`?

Reply:
Yup, done!